src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -55,6 +56,24 @@ public OpenIdConnectOptions(string authenticationScheme)
             Events = new OpenIdConnectEvents();
             Scope.Add("openid");
             Scope.Add("profile");
+
+            ClaimMaps.DeleteClaim("nonce");
```

Comment:
What's the rationale against params?  We certainly have them in several places today already for exactly this situation, truncating a lot of duplicated lines, maybe you don't even really need DeleteClaim, just a single params DeleteClaims